modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/QuickActions.tsx

```diff
@@ -26,7 +25,7 @@ function QuickActions({actions, itemData, itemId, onClick}) {
 						}
 						key={action.data?.id || action.label}
 						monospaced={false}
-						onClick={(event) =>
+						onClick={(event: any) =>
```

Comment:
There are quite a few `any` types in this PR. These are not ideal, most if not all of these can be improved, but we can leave that for followups. Just the initial conversion to TS is a big enough step.